packit_service/models.py

```diff
@@ -151,6 +151,11 @@ class CoprBuild(Base):
     web_url = Column(String)
     # url to copr build logs
     build_logs_url = Column(String)
+    # time.time instead of time.time() because its an argument to the function
+    # so it will run when the copr build is initiated, not when the table is made
+    build_submitted_time = Column(Integer, default=time.time)
+    build_start_time = Column(Integer)
+    build_finished_time = Column(Integer)
```

Comment:
what do others think? int or datetime? I personally prefer datetime, I can see that SA supports DT natively https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.DateTime

Reply:
If there is no big issue with `datetime`, I would go with that.

Reply:
I already said that datetime is perfect.
I had initially suggested timestamp because it's IMHO better than storing string.

Reply:
@jpopelka  Switched back to datetime in the latest commit. In redis, I'm currently storing the same datetime object as a string, but I think it can be converted back into a datetime object.

Reply:
and we're soon moving away from redis, so I wouldn't worry about it too much